### DISCUSSION TITLE
Member list in Organization not loading

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [dani-garcia/vaultwarden#4212] [Q&A] [answered]
I have added a new user to my self hosted environment and when I went to add them to my existing organization (that has 3 users already). I noticed on the web under Organization > Members it just lists the spinning dot wheel and nobody is listed. 

Now I am doing this remotely via vpn so maybe something is blocking it from the vpn but everything else seems to work. 

Any thoughts/ideas?

![IMG_0400](https://github.com/dani-garcia/vaultwarden/assets/68966864/f6844469-968b-426c-befa-25e4a23899a1)

**Comment by BlackDex** [comment]
It looks like the web-vault is not currently loaded. Might be reverse proxy cache or browser cache.

**Reply by Lunchtime0614** [reply]
If I go to the vault section it loads - all my logins and such load fine...is that what you mean by web-vault?

It is behind cloudflare zero trust....but that's not enabled since I'm within my network via the vpn.  Is there something I should do to clear the cache?

**Comment by stefan0xC** [comment, marked as answer]
Can you check the browser console and also server logs when opening the members tab? Also if you had this instance for a long time you might want to check under settings if the organization has a valid email address (e.g. `a@b`) otherwise the loading will fail.

**Reply by Lunchtime0614** [reply]
Interesting - it doesn't have a valid email anymore...and when I enter one, it reverts to only part of my email I entered and says the what I entered is not valid.

**Reply by Lunchtime0614** [reply]
I had to log out and back in and then it seemed to take my email...and then the member list loaded.  I still think there might be some issues with how I have zero trust setup in cloudflare as I'm seeing some things blocked in the console but it seems to be working now.

**Comment by Lunchtime0614** [comment]
I just opened the console and the network tab - something from cloudflare is being blocked - some sort of session is being blocked, odd that it lets me get to everything else within vaultwarden web.
